src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -156,7 +304,13 @@ private static void SetPreciseMachineCounters(IEnumerable<ProviderInfo> provider
                 }
 
                 if (profileSourceIDs.Count > 0)
+                {
+                    // 
+                    // FIXME: This function changes the -pmcsources intervals machine wide.
```

Comment:
Yes, we should fix this in this PR.  I don't think we want machine-wide state changes not getting cleaned up.